server/routes/referrals/serviceUserDetailsPresenter.ts

```diff
@@ -12,14 +12,25 @@ export default class ServiceUserDetailsPresenter {
 
   readonly title = `${this.serviceUser.firstName || 'Service user'}'s information`
 
-  get summary(): SummaryListItem[] {
-    // required to force type erasure of type null[] from list of (string[] | null[])
-    function notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
-      return value !== null && value !== undefined
-    }
-    const phoneNumbers: string[] | undefined = this.deliusServiceUserDetails.contactDetails.phoneNumbers
+  // required to force type erasure of type null[] from list of (string[] | null[])
+  private notEmpty<TValue>(value: TValue | null | undefined): value is TValue {
```

Comment:
I've found this function a bit hard to parse, but it's probably because I'm not good enough at Typescript - I don't suppose there's some way we can make it more readable, is there? I think it's the name `TValue` that's confusing me.

Reply:
TValue is for the generic Type T, I'll change it to String since we're dealing with Strings

Reply:
Thanks!

Reply:
done